app/models/user.rb

```diff
@@ -27,4 +27,9 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :calendars, foreign_key: "owner_id"
+  has_many :events, through: :calendarsI
```

Comment:
It looks like there's a little typo here! 😵‍💫

Reply:
resolved!